ROLES.md

```diff
@@ -126,51 +91,52 @@ issues and PRs assigned to them, participate in working group meetings, and
 pre-submit tests are automatically run for their PRs. Members are expected to
 remain active contributors to the community.
 
-All members are encouraged to help with the code review burden, although each PR
-must be reviewed by an official [Approver](#approver).
+All members are encouraged to help with code reviews, although each PR
+must be reviewed and approved by an official [Approver](#approver).
 
 When reviewing, members should focus on code quality and correctness, including
 testing and factoring. Members might also review for more holistic issues, but
 this is not a requirement.
 
 ### Requirements
 
-- Member of the Knative github org. Create a PR adding you to
+- Member of the Knative github org. Create a PR adding yourself as a Member to
 [knative.yaml](../main/peribolos/knative.yaml) and/or to
-[knative-sandbox.yaml](../main/peribolos/knative-sandbox.yaml) which after when
-merged will send you an invite that you have to accept to become a member
-of these organizations.
+[knative-sandbox.yaml](../main/peribolos/knative-sandbox.yaml), and as a `reviewer`
+on the repositories to which you will be contributing. After the PR is
+merged you will receive an invite that you must accept to become a Member
+of these GitHub organizations.
 
 - Has made multiple contributions to the project or community. Contributions
   might include, but are not limited to:
 
-  - Authoring and reviewing PRs on GitHub.
-
-  - Filing and commenting on issues on GitHub.
-
-  - Contributing to working group or community discussions.
+  - Authoring and reviewing PRs on GitHub
+  - Submitting and commenting on issues on GitHub
+  - Contributing to working group or community discussions
 
 - Subscribed to
-  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev).
+  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev)
+
+- Actively contributing to 1 or more areas
 
-- Actively contributing to 1 or more areas.
+- Nominated by two Knative Members, at least one of whom does not work for the same employer
```

Comment:
What's the motivation behind adding this requirement?  Is this to prevent ballot stuffing?  If so, we might want to put in the 50 contributions in the last year bar, rather than the nomination bar.

(In particular, we've been quite free with org member privileges in the past, and if we want to set this new bar, we may need to more actively clean up our org membership.)

Reply:
In fact, I'd be in favor of keeping the bar for org membership low enough that wanting it is enough, unless someone is clearly not acting in the best interests of the project.  But I'd be interested in seeing what experience @geekygirldawn and @jberkus have with setting thresholds on org membership.

Reply:
This was also added based on the CNCF template. I don't have strong feelings about it either way. I think at least one nomination might be a good idea however, as this would be a tangible thing that a Working Group Lead, Member, or Approver could do to help promote the community. It could, in fact, be a Responsibility for Members and Approvers as a recognized contribution to the project.

Reply:
I'm also a fan of keeping the bar to org membership low. Being nominated by 2 people probably comes from K8s, and Knative is a smaller project, so a lower bar would be fine. You could probably just remove the nomination requirement, and rely on the person merging the PR to make sure that it's not spam / malicious.